test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,209 @@
+    dc2_connection = manager.connect_to_node(servers[1])
+    random_tables = RandomTables(request.node.name, manager, ks, 1, dc_replication)
+    await random_tables.add_table(ncolumns=2, columns=columns, pks=1, name=table_name)
+    dc1_connection.execute(
+        f"INSERT INTO  {ks}.{table_name} ({columns[0].name}, {columns[1].name}) VALUES ('{expected[0]}', '{expected[1]}');")
+    query_cl_one = SimpleStatement(f"SELECT * from {ks}.{table_name};", consistency_level=ConsistencyLevel.ONE)
+    query_cl_local_quorum = SimpleStatement(f"SELECT * from {ks}.{table_name};",
+                                            consistency_level=ConsistencyLevel.LOCAL_QUORUM)
+    nodetool.flush(dc1_connection, "{ks}.{table_name}")
+    first_node_results = list(dc1_connection.execute(query_cl_one).one())
+    second_node_result = dc2_connection.execute(query_cl_local_quorum).one()
+
+    assert first_node_results == expected, \
+        f"Expected {expected} from {query_cl_one.query_string}, but got {first_node_results}"
+    assert second_node_result is None, \
+        f"Expected no results from {query_cl_local_quorum.query_string}, but got {second_node_result}"
+
+
+def _put_with_overwrite(session, nb_keys, cf, ks, cl=ConsistencyLevel.QUORUM):
+    """
+    Put 100 rows with for specified keys. Then overwrite partially existing rows with new values
+
+    :param session: Session instance
+    :param nb_keys: Number of keys to add
+    :param cf: Column family
+    :param ks: Key space
+    :param cl: Consistency level
+    """
+    update_query = f"UPDATE {cf} SET v='value%d' WHERE key='k%s' AND c='c%02d'"
+    query_batch = "BEGIN BATCH %s APPLY BATCH"
+    for k in range(nb_keys):
+        kvs = [update_query % (i, k, i) for i in range(100)]
+        query = SimpleStatement(
+            query_batch % ";".join(kvs), consistency_level=cl
+        )
+        session.execute(query)
+    nodetool.flush_keyspace(session, ks)
+    for k in range(nb_keys):
+        kvs = [update_query % (i * 4, k, i * 2) for i in range(50)]
+        query = SimpleStatement(
+            query_batch % "; ".join(kvs), consistency_level=cl
+        )
+        session.execute(query)
+    nodetool.flush_keyspace(session, ks)
+    for k in range(nb_keys):
+        kvs = [update_query % (i * 20, k, i * 5) for i in range(20)]
+        query = SimpleStatement(
+            query_batch % "; ".join(kvs), consistency_level=cl
+        )
+        session.execute(query)
+    nodetool.flush_keyspace(session, ks)
+
+
+def putget(session, cf, ks, cl=ConsistencyLevel.QUORUM):
+    _put_with_overwrite(session, 1, cf, ks, cl)
+
+    query = SimpleStatement(f"SELECT * FROM {cf} WHERE key='k0'", consistency_level=cl)
+    rows = list(session.execute(query))
+    _validate_row(rows)
+
+
+def _validate_row(res):
+    """This method checks final state of the table after overwrites done in :py:_put_with_overwrite():"""
+    assert_length_equal(res, 100)
```

Comment:
Adding this assert_length_equal convenience function  is completely unnecessary and makes it harder to read the code. The idiomatic pytest way to do this is simply:
```
    assert len(res) == 100
```
And that's it! No need for an error string (assertion failures are very clear in pytest) and definitely not for a wrapper function for this assert.

Reply:
Done